cpp/ql/lib/qlpack.yml

```diff
@@ -1,7 +1,8 @@
 name: codeql/cpp-all
-version: 0.0.2
+version: 0.0.3
```

Comment:
Do you want to go with the `-dev` prefix here?

Reply:
I'd like to hold off on that until after our first release. The post-release tool can add that automatically.